examples/3d/shadow_biases.rs

```diff
@@ -152,7 +152,7 @@ fn toggle_light(
         for mut light in &mut point_lights {
             light.intensity = if light.intensity == 0.0 {
                 *writer.text(*example_text, 4) = "PointLight".to_string();
-                100000000.0
+                light_consts::lumens::VERY_LARGE_CINEMA_LIGHT
```

Comment:
VERY_LARGE_CINEMA_LIGHT is 1_000_000 not 100_000_000, does this work well?

Reply:
Yup, its extremely blown-out on main, seems like it was left behind during some unit change. This looks more normal

Reply:
Really the PointLight and DirectionalLight should probably both be default, and one or the other should be Disabled by the cycler, but i dont think that works yet

Reply:
One way or another, the PointLight and DirectionalLight match much more closely on main than in this PR. Both with the initial values on main and also the values that are used after switching.